### ISSUE TITLE
Failed to fetch dynamically imported module: DiaryAddNote.fbf168ef.js

### ISSUE THREAD

**Issue by sigilante** [tloncorp/landscape-apps#1853]
`Add Note` usage fails repeatedly on a notebook I have perms to write in.

I am on ~binnec-dozzod-marzod.

```
TypeError: Failed to fetch dynamically imported module: http://100.105.215.33:8080/apps/groups/assets/DiaryAddNote.fbf168ef.js
```

**Comment by jamesacklin** [comment]
Should be fixed by #1829, but @patosullivan can you verify?

**Comment by patosullivan** [comment]
This is actually should be fixed by deprecating janeway for releases and the work in #1829. Dupe of #1733.
